pkg/kv/kv.go

```diff
@@ -57,6 +57,8 @@ type Iter interface {
 	Value() []byte
 	// Close close this iter.
 	Close() error
+	// KeyIsDelete
+	KeyIsDelete() bool
```

Comment:
Shall we replace this with something like `OpType()` so if we need to add more types of keys e.g. `Merge`, we needn't add another method.

Reply:
@kennytm I want to know what's your thoughts.

Reply:
+1 for glorv.

Reply:
Done, PTAL~